frontend/content/posts/build-your-own-little-aws-console-in-cdk-with-buttonize/index.yml

```diff
@@ -0,0 +1,10 @@
+title: "Build your own little AWS Console in CDK with Buttonize"
+excerpt: "Buttonize helps developers and companies using AWS solve the painful problem of internal tooling. Internal tooling is a perennial problem. We all have these "temporary" tools in our companies that have been around forever with so much tech debt that everybody is afraid to touch the code."
```

Comment:
I believe the double quotes inside of double quotes is causing a failure. Can you try fixing the build by either escaping the "temporary" or by switching to single quotes?

Reply:
True, sorry, didn't notice.